helm/easeprobe/Chart.yaml

```diff
@@ -3,7 +3,7 @@ annotations:
 apiVersion: v2
 appVersion: "2.0.0"
 name: easeprobe
-version: "1.0.0"
+version: "1.0.1"
```

Comment:
It would be good to include a changelog in the Chart.yaml or in a separate CHANGELOG.md file. This would help users and developers track what changes have been made in each version. [medium]